doc/modules/linear_model.rst

```diff
@@ -122,7 +128,7 @@ its ``coef_`` member::
     >>> reg.fit([[0, 0], [0, 0], [1, 1]], [0, .1, 1])
     Ridge(alpha=0.5)
     >>> reg.coef_
-    array([0.34545455, 0.34545455])
+    array([0.34545..., 0.34545...])
```

Comment:
Note: this was not actually needed to make the docstest pass: this PR does not change the Ridge implementation. But I think it makes the docstring more readable and consistent with the number of digits we display for the intercept.